core/pom.xml

```diff
@@ -39,7 +39,11 @@
 
   <properties>
     <jclouds.osgi.import>*</jclouds.osgi.import>
-    <jclouds.osgi.export>org.jclouds*;version=${project.version};-noimport:=true</jclouds.osgi.export>
+    <!-- https://issues.apache.org/jira/browse/JCLOUDS-1529 -->
+    <jclouds.osgi.export>
+      !org.jclouds.json.gson.internal*,
```

Comment:
[nit] Please add a comment that this package is already exported by `jclouds-gson`.